sources/AppBundle/Accounting/Model/Rule.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AppBundle\Accounting\Model;
+
+use CCMBenchmark\Ting\Entity\NotifyProperty;
+use CCMBenchmark\Ting\Entity\NotifyPropertyInterface;
+
+class Rule implements NotifyPropertyInterface
+{
+    use NotifyProperty;
+
+    private ?int $id = null;
+
+    private ?string $label = null;
+    private ?string $condition = null;
+
+    private ?bool $isCredit = null;
+
+    private ?string $vat = null;
+
+    private ?int $categoryId = null;
+
+    private ?int $eventId = null;
+
+    private ?int $paymentTypeId = null;
+
+    private ?bool $attachmentRequired = null;
+
+    public function getId(): ?int
+    {
+        return $this->id;
+    }
+
+    public function setId(int $id): self
+    {
+        $this->propertyChanged('id', $this->id, $id);
+        $this->id = $id;
+        return $this;
+    }
+
+    public function getLabel(): ?string
+    {
+        return $this->label;
+    }
+
+    public function setLabel(?string $label): self
+    {
+        $this->propertyChanged('name', $this->label, $label);
```

Comment:
```suggestion
        $this->propertyChanged('label', $this->label, $label);
```

Il faudrait plutôt mettre le label ici sinon ça cause l'erreur lors de l'édit d'une règle.